changelog.d/pr-8023

```diff
@@ -6,6 +6,6 @@ description: {
 
 - Make builds that use experimental Cabal language features less noisy. At -v1
   (normal) we show just first instance of use of experimental cabal language
-features, along with count of further occurences in the same file.
+features, along with count of further occurrences in the same file.
 
 }
```

Comment:
Please do not touch past changelogs

Reply:
https://github.com/haskell/ThreadScope/pull/123#issuecomment-1105604564